src/v2/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx

```diff
@@ -1,7 +1,6 @@
 import React from "react"
 import { SettingsEditProfileRoute_me } from "v2/__generated__/SettingsEditProfileRoute_me.graphql"
 import { createFragmentContainer, graphql } from "react-relay"
-import { UserInformationRefetchContainer } from "./UserInformation"
```

Comment:
This had no business being a refetch container. I think someone just misunderstood that you can just fetch attrs in the mutation? I'm not sure.